server/utils/llm.py

```
@@ -381,3 +381,38 @@ async def call_llm_messages_async(
+            max_tokens=max_tokens,
+            stream=True,
+        )
+        async for chunk in stream:
+            if chunk.choices and chunk.choices[0].delta is not None:
+                content = chunk.choices[0].delta.content
+                if content:
+                    yield content
+    except APIError as e:
+        logger.exception("LLM stream failed")
+        raise ValueError("LLM stream failed") from e
+    except Exception as e:
```

Ruff (0.14.14)

[error] Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)